Kinvey/Kinvey/MIC.swift

```diff
@@ -361,7 +362,7 @@ public enum MICApiVersion: String {
 import UIKit
 import WebKit
 
-class MICLoginViewController: UIViewController, WKNavigationDelegate, UIWebViewDelegate {
+class MICLoginViewController: UIViewController {
```

Comment:
Is this only used internally, or could this be a breaking change for developers?

Reply:
that's an internal class, so, no breaking changes at this point, but I end up deciding to remove UIWebView completely because of the tests were not working, meaning, keeping the `MICUserInterface.uiWebView` could cause bugs so, we will have to make a major release `4.0`